### PR TITLE
Get presentations

### PR DESCRIPTION
**Pull request by Jasonn5**
image

### CODE REVIEW

File: config.py
Comment (FreddyFZ17) [inline config.py:1]: I have no observations, and it seems to me that the dictionary fulfills its functionality.

File: src/assertions/assertion_comparison.py
Comment (FreddyFZ17) [inline src/assertions/assertion_comparison.py:1]: Both functions serve their purpose clearly and are easily understandable. They are useful tools for checking critical conditions in the code and I can't find any observations in this regard.

File: src/espocrm_api/meetring_endpoints.py
Comment (FreddyFZ17) [inline src/espocrm_api/meetring_endpoints.py:1]: I have no observations and I consider that it is well implemented following what was established as a team.

File: src/resources/auth/auth.py
Comment (FreddyFZ17) [inline src/resources/auth/auth.py:1]: This code in the Auth class handles user authentication by loading credentials from a JSON file. It provides clear methods to verify different authentication cases, improving security and flexibility in managing users within the application.

File: tests/login/test_login.py
Comment (FreddyFZ17) [inline tests/login/test_login.py:1]: Improvements have been implemented using the new resources provided for login and authentication.

File: tests/presentations/test_GetPresentations.py
Comment (FreddyFZ17) [inline tests/presentations/test_GetPresentations.py:1]: New test cases were added and existing ones were improved using the new resources agreed upon as a team. Overall, I find the implementation to be correct. I just want to highlight that the test_get_meetings_invalid_param testcase executed incorrectly, but it revealed a bug, which has been noted, so I have no further comments.

**Review by FreddyFZ17 — APPROVED** [review]

**Review by FreddyFZ17 — APPROVED** [review]
In general, I have no observations and I think everything is well implemented, since I had no issues with its execution and it was very easy to understand, except for the fact that a bug was found.

File: config.py
Comment (AlisonGuzman2006) [reply config.py:1]: I have no observations, the changes made still maintains the functionality, and allow code to be refined

File: src/espocrm_api/meetring_endpoints.py
Comment (AlisonGuzman2006) [reply src/espocrm_api/meetring_endpoints.py:1]: I have no observations

File: src/resources/auth/auth.py
Comment (AlisonGuzman2006) [reply src/resources/auth/auth.py:1]: I've reviewed the proposed changes and am happy to say that everything looks good. The new functionality works as expected and the code is clean.

File: src/assertions/assertion_comparison.py
Comment (AlisonGuzman2006) [reply src/assertions/assertion_comparison.py:1]: Both assert methods are understandable and can be used by the whole team.

File: tests/login/test_login.py
Comment (AlisonGuzman2006) [reply tests/login/test_login.py:1]: I've reviewed the changes made to the resources_schemas_path and resources_credential_path functions, as well as the modifications in tests/login/test_login.py. The refactoring looks clean and enhances the maintainability of the code.

File: tests/presentations/test_GetPresentations.py
Comment (AlisonGuzman2006) [reply tests/presentations/test_GetPresentations.py:1]: I've reviewed the updates to the test cases in tests/login/test_login.py, and the changes are well-structured and maintain the integrity of our login functionality tests.

**Review by AlisonGuzman2006 — APPROVED** [review]
I aprove the changes, and I am happy the code being refractored and have maintainability

**Pull request merged**